### PR TITLE
Django2.2

### PR DESCRIPTION
**Pull request by domdinicola**

### CODE REVIEW

File: data_api/staging/models.py [hunk 1]
```diff
@@ -347,6 +347,9 @@ class Runs(RapidproBaseModel):
     expired = models.IntegerField(default=0)
     interrupted = models.IntegerField(default=0)
 
+    def __unicode__(self):
```
Comment (czue) [inline data_api/staging/models.py]: should this be `__str__`?

Comment (domdinicola) [reply data_api/staging/models.py]: sure!

**Comment by domdinicola** [comment]
@czue ready!

**Comment by domdinicola** [comment]
@czue can you look at it?

**Comment by czue** [comment]
yes! sorry looking and testing now.

**Comment by domdinicola** [comment]
@czue now should be fixed..
since drf 1.10 https://www.django-rest-framework.org/community/3.10-announcement/#continuing-to-use-coreapi
maybe we will move to openAPI in the future

**Comment by czue** [comment]
Looks good!

**Pull request merged**